### PR TITLE
Fix bug with wrong kwargs. Bump version

### PR DESCRIPTION
**Pull request by anmaxvl**

### CODE REVIEW

File: gdc_client/version.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = 'v1.3.0'
+__version__ = 'v1.3.1'
```
Comment (ProfOak) [inline gdc_client/version.py]: This is probably ok for now but we don't normally bump version numbers until we're ready to do a release.

File: gdc_client/defaults.py [hunk 2]
```diff
@@ -34,3 +34,6 @@
 proxy_port = 9000
 
 HTTP_CHUNK_SIZE = 1024 * 1024 * 1024  # 1 GiB
+
+# Bringing this back to avoid errors in other modules
+part_size = HTTP_CHUNK_SIZE
```
Comment (ProfOak) [inline gdc_client/defaults.py]: Where else uses this?

File: gdc_client/version.py (hunk 1 above)
Comment (anmaxvl) [reply gdc_client/version.py]: Ah ok. This is something that Ian asked me to do for his build. Will put it back.

File: gdc_client/defaults.py (hunk 2 above)
Comment (anmaxvl) [reply gdc_client/defaults.py]: Just double checked, seems to be nowhere else. But `repl` was still using it and I already changed it to `HTTP_CHUNK_SIZE`

**Review by ProfOak — APPROVED** [review]

**Pull request merged**